internal/controller/siteconfig_controller.go

```diff
@@ -32,7 +35,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 
-	siteconfigv1alpha1 "github.com/sakhoury/siteconfig/api/v1alpha1"
+	sc_v1alpha1 "github.com/sakhoury/siteconfig/api/v1alpha1"
```

Comment:
FWIW for the local API I usually leave it unprefixed so this could just be `v1alpha1` then everything else would have a prefix (`metav1`, for example)

Reply:
I like this a lot! I'll adopt the same practice. Changed to `v1alpha1`.